source/content/guides/wordpress-composer/06-wordpress-ic.md

```diff
@@ -0,0 +1,97 @@
+---
+title: WordPress with Integrated Composer on Pantheon
+subtitle: Use Integrated Composer with WordPress 
+description: Learn how to use Integrated Composer with WordPress on Pantheon.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/wordpress-ic
+anchorid: wordpress-ic
+---
+
+[Integrated Composer](/guides/integrated-composer) is a Pantheon platform feature. Integrated Composer extends Composer functionality to WordPress core files and treats them as a managed dependency. Integrated Composer lets you deploy your site on Pantheon with one-click updates for both upstream commits and Composer dependencies, while still receiving upstream updates.
+
+## WordPress with Integrated Composer on Pantheon 
+
+Pantheon maintains a [Composer-enabled WordPress Upstream](https://github.com/pantheon-upstreams/wordpress-project) repository. This is the recommended starting point for forking new upstreams that work with Pantheon's Integrated Composer build process.
+
+This repository is still under active development, and you should not create permanent/production sites from it. There is no guarantee of backwards compatibility. 
+
+Pantheon upstreams influence the Composer packages that are included in downstream sites by including two `composer.json` files in this repository:
+
+- The root `/composer.json` file is owned by the downstream site. Upstream maintainers should avoid editing this file. This allows the downstream site maintainer to adjust the `/composer.json` file without creating potential conflicts when merging upstream updates.
+
+- The `upstream-config/composer.json` file is owned by the upstream maintainer. It is included by the root `composer.json` file, and allows upstreams to add or remove packages from downstream sites. Changes are automatically incorporated into the downstream site whenever upstream updates are applied.
+
+## Create Your WordPress Integrated Composer Site
+
+Follow the steps in this section to create a new WordPress site using Integrated Composer and Upstream.
+
+1. Fork the [Pantheon-maintained repository](https://github.com/pantheon-upstreams/wordpress-project).
+
+1. [Connect your repository](/create-custom-upstream#connect-repository-to-pantheon) to Pantheon.
+
+1. Edit the **vendor** name used in the `upstream-config` directory.
```

Comment:
Is this really needed? I think there is no harm in leaving it as "pantheon-upstreams" and it will make easier to adopt possible updates in their custom upstreams in the future